opba-banking-protocol-facade/src/main/java/de/adorsys/opba/protocol/facade/config/EncryptionConfig.java

```diff
@@ -6,27 +6,28 @@
 import com.google.crypto.tink.JsonKeysetReader;
 import com.google.crypto.tink.KeysetHandle;
 import com.google.crypto.tink.aead.AeadConfig;
-import lombok.AllArgsConstructor;
 import lombok.SneakyThrows;
 import org.bouncycastle.jce.provider.BouncyCastleProvider;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
 
 import java.nio.file.Paths;
 import java.security.SecureRandom;
 import java.security.Security;
 
 @Configuration
-@AllArgsConstructor
 public class EncryptionConfig {
     public static final SecureRandom SECURE_RANDOM = new SecureRandom();
-    private final EncryptionProperties properties;
+    @Value("${facade.encryption.providerName:BC}") String providerName;
+    @Value("${facade.encryption.keySetPath:example-keyset.json}") String keySetPath;
 
     @Bean
     @SneakyThrows
+    @Profile("!no-enc")
```

Comment:
Please rename to `no-encryption` and add it to some Const utility class

Reply:
ok